stacks/HANA-Scaleout-Standby/playbook-notf.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: SAP HANA scaleout master and worker configure
+  hosts: hana
+  become: true
+  vars:
+    sap_hana_backint_install: '{{ sap_hana_backint_bucket_name | default("") != "" }}'
+    sap_hana_backint_bucket_name: '{{ sap_hana_backint_bucket_name | default("") }}'
+    sap_hana_worker_disks_list: '{{ terraform.outputs.worker_attached_disks_data.value }}'
+    sap_hana_data_size: '{{ terraform.outputs.hana_data_size.value }}G'
```

Comment:
There are no terraform outputs here to retrieve these from. The vars can come from files or inventory.

Reply:
There are outputs defined for hana disk sizes in the outputs.tf for the stack.

Reply:
The purpose of `playbook-notf.yml` is to use Terraform separately from Ansible or not at all. Since the forminator role has not been run, there would not be a `terraform` variable in the playbook to get outputs from. These variables would need to be defined by the user and put in a vars.yml or similar.

Reply:
Agreed. Just pushed the change